src/steem/rc.ts

```diff
@@ -0,0 +1,49 @@
+import { Bignum } from "./misc";
+import { Nai, PriceType } from "./asset"
+
+export interface RCParams {
+    resource_history_bytes: Resource,
+    resource_new_accounts: Resource,
+    resource_market_bytes: Resource,
+    resource_state_bytes: Resource,
+    resource_execution_time: Resource,
+}
+export interface Resource {
+    resource_dynamics_params: DynamicParam,
+    price_curve_params: PriceCurveParam
+}
+export interface DynamicParam {
+    resource_unit: number,
+    budget_per_time_unit: number,
+    pool_eq: string | number,
+    max_pool_size: string | number,
+    decay_params: {
+        decay_per_time_unit: Bignum,
+        decay_per_time_unit_denom_shift: number
+    },
+    min_decay: number
+}
+export interface PriceCurveParam {
+    coeff_a: string | number,
+    coeff_b: string | number, //could be Bignum
```

Comment:
Yep, all string | number returns from steemd are `Bignum`

Reply:
ah ok, changing that then.